database/migrations/2022_07_16_083805_add_color_column_in_table_categories.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddColorColumnInTableCategories extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('categories', function (Blueprint $table) {
+            $table->string('color')->after('type')->default('badge-gray');
```

Comment:
can we have it just gray as default?
```suggestion
            $table->string('color')->after('type')->default('gray');
```